pom.xml

```diff
@@ -314,6 +314,12 @@
       <artifactId>simpleclient_servlet</artifactId>
       <version>0.5.0</version>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/org.jgroups/jgroups-raft -->
+    <dependency>
```

Comment:
Not directly related to this change, but I find it frustrating that maxwell has so many dependencies, lots of which are not used in all use cases.  For those of us that have enterprise tools to do vulnerability scanning, it creates noise with dependencies that have vulnerabilities which we aren't even using.  I'd love to see maxwell become more modular (but appreciate that is a lot of effort).